parchmint/feasibilitymatcher.py

```diff
@@ -0,0 +1,70 @@
+from os import truncate
+from typing import Dict
+from networkx.algorithms.isomorphism import DiGraphMatcher
+from networkx.classes import digraph
+# from reggie.nodefilter import NodeFilter
+# from lfr.fig.fluidinteractiongraph import FluidInteractionGraph
+
+
+class FeasibilityMatcher(DiGraphMatcher):
+    """Implementation of VF2 algorithm for matching undirected graphs.
+    Suitable for Graph and MultiGraph instances.
+    """
+
+    def __init__(
+        self,
+        G1: digraph.DiGraph,
+        G2: digraph.DiGraph,
+        # semantic_information: Dict[str, NodeFilter],
+    ):
+        # self._semantic_information = semantic_information
+        super(FeasibilityMatcher, self).__init__(G1, G2)
```

Comment:
Extract the graphs from the device objects and then do the super the way you're doing this